conf/distro/iot2050-debian-qemu.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Enes �olpan <enes.colpan@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require conf/distro/iot2050-debian.conf
+
+# used for SWUpdate boot
+ABROOTFS_IMAGE_RECIPE = "iot2050-image-qemu"
+VERITY_IMAGE_RECIPE = "iot2050-image-qemu"
```

Comment:
Do we really need an own image, or can we also get away with just an own machine?

Reply:
Switched to machine instead of image.